tools/topology/topology2/sof/sof-hda-generic.conf

```diff
@@ -0,0 +1,455 @@
+			index 9
+			type dai_in
+			stream_name iDisp3
+			period_sink_count 0
+			period_source_count 2
+			format s32le
+		}
+		direction playback
+	}
+}
+
+# Pipeline Definitions
+Object.Pipeline {
+	volume-playback.1 {
+		dynamic_pipeline $DYNAMIC_PIPELINE
+		index 1
+		Object.Widget.pipeline.1 {
+			stream_name 'dai.HDA.0.playback'
+		}
+		Object.Widget.host.playback {
+			stream_name 'Passthrough Playback 0'
+		}
+		Object.Widget.pga.1 {
+			Object.Control.mixer.1 {
+				name '1 Master Playback Volume'
+			}
+		}
+		format s24le
+	}
+	volume-playback.3 {
+		dynamic_pipeline $DYNAMIC_PIPELINE
+		index 3
+		Object.Widget.pipeline.1 {
+			stream_name 'dai.HDA.2.playback'
+		}
+		Object.Widget.host.playback {
+			stream_name 'Passthrough Playback 1'
+		}
+		Object.Widget.pga.1 {
+			Object.Control.mixer.1 {
+				name '3 Master Playback Volume'
+			}
+		}
+		format s24le
+	}
+	volume-playback.7 {
+		dynamic_pipeline $DYNAMIC_PIPELINE
+		index 7
+		Object.Widget.pipeline.1 {
+			stream_name 'dai.HDA.4.playback'
+		}
+		Object.Widget.host.playback {
+			stream_name 'Passthrough Playback 3'
+		}
+		Object.Widget.pga.1 {
+			Object.Control.mixer.1 {
+				name '7 Master Playback Volume'
+			}
+		}
+		format s24le
+	}
+	volume-playback.8 {
+		dynamic_pipeline $DYNAMIC_PIPELINE
+		index 8
+		Object.Widget.pipeline.1 {
+			stream_name 'dai.HDA.5.playback'
+		}
+		Object.Widget.host.playback {
+			stream_name 'Passthrough Playback 4'
+		}
+		Object.Widget.pga.1 {
+			Object.Control.mixer.1 {
+				name '8 Master Playback Volume'
+			}
+		}
+		format s24le
+	}
+	volume-playback.9 {
+		dynamic_pipeline $DYNAMIC_PIPELINE
+		index 9
+		Object.Widget.pipeline.1 {
+			stream_name 'dai.HDA.6.playback'
+		}
+		Object.Widget.host.playback {
+			stream_name 'Passthrough Playback 5'
+		}
+		Object.Widget.pga.1 {
+			Object.Control.mixer.1 {
+				name '9 Master Playback Volume'
+			}
+		}
+		format s24le
+	}
+	eq-iir-volume-capture.2 {
+		dynamic_pipeline $DYNAMIC_PIPELINE
+		index 2
+		Object.Widget.pipeline.1 {
+			stream_name 'dai.HDA.1.capture'
+		}
+		Object.Widget.host.capture {
+			stream_name 'Highpass Capture 0'
+		}
+		Object.Widget.pga.1 {
+			Object.Control.mixer.1 {
+				name '2 Master Capture Volume'
+			}
+		}
+		Object.Widget.eqiir.1 {
+			Object.Control.bytes.1 {
+				name eqiir_bytes_2
+				# EQIIR filter coefficients
+				IncludeByKey.EQIIR_BYTES {
+					"[highpass.40hz.0db.48khz]" "include/components/eqiir/highpass_40hz_0db_48khz.conf"
+					"[highpass.40hz.20db.48khz]" "include/components/eqiir/highpass_40hz_20db_48khz.conf"
+				}
+			}
+		}
+		format s24le
+	}
+	eq-iir-volume-capture.4 {
+		dynamic_pipeline $DYNAMIC_PIPELINE
+		index 4
+		Object.Widget.pipeline.1 {
+			stream_name 'dai.HDA.3.capture'
+		}
+		Object.Widget.host.capture {
+			stream_name 'Highpass Capture 1'
+		}
+		Object.Widget.pga.1 {
+			Object.Control.mixer.1 {
+				name '4 Master Capture Volume'
+			}
+		}
+		Object.Widget.eqiir.1 {
+			Object.Control.bytes.1 {
```

Comment:
Do these Object.Control.bytes.N values be incremented? I tried to setup 2 and 4 to use different coefficients. The eq_iir_bytes_2 and eq_iir_bytes_4 referred to same coefficient in data [].